app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MessagesController < ApplicationController
+
+  def create
+    @message = Message.new(message_params)
+    @match = Match.find_by(id: params[:match_id])
+    @message.user = current_user
+    @message.match = @match
+    @my_item = current_user.items.first
+    if @message.save
+      @match.touch
```

Comment:
You can put the touch: true on the belongs_to if you always want saving a message to touch the match.